src/odemis/gui/cont/acquisition.py

```diff
@@ -1573,6 +1575,11 @@ def on_acquisition_done(self, future):
             self._reset_acquisition_gui("Acquisition failed (see log panel).", level=logging.WARNING)
             return
 
+    def _on_storage_dir(self, sdir):
+        sdir = sdir.split('@')[-1]  # remove username/password
+        sdir = sdir.split('/', maxsplit=1)[-1]  # remove host
+        self._tab_panel.txt_destination.SetValue("%s/%s" % (sdir, self.path))
```

Comment:
Probably the setting of the external storage is already logged somewhere else. But in the case it isn't I think at least a debug/info log should be made.